### ISSUE TITLE
[Idea]: Use prisma serverless adapter instead of `@planetscale/database` directly

### ISSUE THREAD

**Issue by pjeweb** [alveusgg/alveusgg#617]
### Describe your idea / enhancement

I think in preparation for #586 we should move the few existing edge routes that use `@planetscale/database` to use prisma instead which now supports serverless driver adapters (e.g. planetscale and vercel db aka neon).

https://www.prisma.io/docs/orm/overview/databases/database-drivers#serverless-driver-adapters

### How would this idea / enhancement be beneficial?

Uniform database access through prisma, help migrate to postgres/vercel db.

**Comment by ACA30** [comment]
@pjeweb is this still needed?

**Comment by pjeweb** [comment]
> @pjeweb is this still needed?
> 
> Seems like no, only things I see still using planetscale are in edge middleware (can't use prisma) or the chatbot (I didn't check the chatbot at all)

Yeah sure, its not a high prio, since we do not immediately plan to switch databases. But not relying on a particular db hosters library seems good in its own.

Also as you pointed out its just used in the edge middleware but it should actually work with Prisma now since they added support for serverless adapters: https://www.prisma.io/docs/orm/overview/databases/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-orm-preview

The chatbot uses drizzle, which i thought at the time of implementing the bot, might've been a replacement for prisma, so i tried it out. I think we'll stick with Prisma for this project though. The concerns around size, speed and serverless adapaters are pretty much alleviated.

**Comment by pjeweb** [comment]
For reference https://www.prisma.io/docs/orm/prisma-client/deployment/edge/deploy-to-vercel
